static/app/views/aiMonitoring/landing.tsx

```diff
@@ -29,6 +34,16 @@ function NoAccessComponent() {
 
 export default function AiMonitoringPage() {
   const organization = useOrganization();
+  const {
+    selection: {projects: selectedProjectIds},
+  } = usePageFilters();
+  const {projects} = useProjects();
+
+  const selectedProjects = projects.filter(p =>
+    selectedProjectIds.find(x => '' + x === p.id)
+  );
+  const isOnboarding =
+    selectedProjects.length > 0 && selectedProjects.every(x => !x.firstTransactionEvent);
```

Comment:
Seems like it's about as good as using `useOnboardingProject`, right?

Reply:
good catch